### ISSUE TITLE
Invalid or unsupported plugin type

### ISSUE THREAD

**Issue by TheJAXi** [falkTX/Carla#1467]
What do I do? It gives this error on every plugin I try (Windows ones). I'm kinda out of options. I kinda need this VST hosting to work on my Linux system so I can route devices to other devices. I'm running Arch linux 5.13.13.

**Comment by falkTX** [comment]
Install the bridges from AUR?

**Comment by TheJAXi** [comment]
I did.

**Comment by TheJAXi** [comment]
The plugins/half of them show up in Carla (how do I get the rest to work?), but I'm unable to enable any of them.

**Comment by TheJAXi** [comment]
Sorry for spamming your thread. I'm kinda in hurry & desperate.

**Comment by falkTX** [comment]
and drag&drop of the dll doesnt work?

**Comment by TheJAXi** [comment]
Yup it says the invalid plugin thing. The plugins work fine with Windows. I also tried the yab thing (can't remember the name), but the copied plugins don't work either. I wonder if it's something to do with my Linux itself. Is there a command I can use to share some information? Like about the app versions & stuff?

**Comment by falkTX** [comment]
carla might not be recognizing the dll as a plugin.
if you run `file /path/to/dll` (so just the file tool) and post the output, I can see if that is why or not.

carla uses the same stuff as the "file" tool to figure out if something is a windows plugin or not, so my guess is that step is failing.

**Comment by TheJAXi** [comment]
/home/jaxi/All VST in One/Brainworx/bx_console SSL 4000 E.dll: PE32+ executable (DLL) (GUI) x86-64, for MS Windows

**Comment by falkTX** [comment]
Well I am afraid I can't help you. The output that you gave me matches the one that Carla looks for, so the issue is not there.
Perhaps your wine setup is just broken?
